ha/setup/ha_setup.py

```diff
@@ -304,11 +307,13 @@ def process(self):
                         # Delete the node from nodelist if it was added in the store
                         if self._confstore.key_exists(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}"):
                             self._confstore.delete(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}")
-
+                nodes = self._confstore.get(f"{const.CLUSTER_CONFSTORE_NODES_KEY}")
```

Comment:
I think this code could be moved to line number 297 above.

Reply:
Done